src/lib/file_system/file_system.ml

```diff
@@ -5,6 +5,15 @@ let remove_dir dir =
   let%bind _ = Process.run_exn ~prog:"rm" ~args:["-rf"; dir] () in
   Deferred.unit
 
+let rec rmrf path =
```

Comment:
This should be about the same as `remove_dir` above, right?

Reply:
Yes, the only difference is that this one doesn't need to be a deferred.